Makefile

```diff
@@ -234,11 +234,13 @@ run-test:
 	export RUST_BACKTRACE=1 && \
 	cargo test --workspace \
 		--exclude fuzzer-honggfuzz --exclude fuzzer-afl --exclude fuzzer-libfuzzer \
-		--features "${ENABLE_FEATURES}" ${EXTRA_CARGO_ARGS} -- --nocapture && \
+		--features "${ENABLE_FEATURES} mem-profiling" ${EXTRA_CARGO_ARGS} -- --nocapture && \
 	if [[ "`uname`" == "Linux" ]]; then \
 		export MALLOC_CONF=prof:true,prof_active:false && \
 		cargo test --features "${ENABLE_FEATURES} mem-profiling" ${EXTRA_CARGO_ARGS} -p tikv_alloc -- --nocapture --ignored && \
-		cargo test --features "${ENABLE_FEATURES} mem-profiling" ${EXTRA_CARGO_ARGS} -p tikv --lib -- -- nocapture --ignored; \
+		cargo test --workspace \
```

Comment:
I'm afraid tests will be run twice.

Reply:
which tests you mean? Can you point it out?

Reply:
I just ran those tests using `make run-test` and didn't see repeated test result.

Reply:
It seems I misunderstood how `--ignored` worked.